guide/14-deep-learning/how_retinanet_works.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
<blockquote>RetinaNet is one of the best object detection model that has proven to work well with dense and small scale objects. For this reason, it has become a popular object detection model to be used with aerial and satellite imagery. RetinaNet has been formed by making two improvements over existing single stage object detection models - Feature Pyramid Networks (FPN)[1] and Focal Loss [2]. Before diving into the exact model architecture of RetinaNet let's first try to understand FPN.</blockquote><p>RetinaNet is one of the best object detection models that has proven to work well with dense and small-scale objects. For this reason, it has become a popular object detection model to be used with aerial and satellite imagery. </p><p> </p><p><em>Comment: Should we have the pools detection image here.</em></p><p> </p><p>RetinaNet has been formed by making two improvements over existing single stage object detection models - Feature Pyramid Networks (FPN) [1] and Focal Loss [2]. Before diving into model’s architecture, let's first understand FPN.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I thought it might be a good idea to have an example image of object detection using RetinaNet. We can change the Figure caption to "Óbject Detection Using RetinaNet" if that makes more sense.</p>

Reply:
@Kapil-varshney Is your comment related to the change suggested above.

Reply:
@Kapil-varshney Is your comment related to the change suggested above.